bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsVoTableCatalog.scala

```diff
@@ -69,7 +69,7 @@ object GemsVoTableCatalog {
   private def searchCatalog(basePosition: Coordinates, criterions: List[GemsCatalogSearchCriterion], statusLogger: StatusLogger): Future[List[GemsCatalogSearchResults]] = {
     val queryArgs = for {
       c <- criterions
-      q = CatalogQuery(basePosition, c.criterion.radiusLimits, c.criterion.magLimits.some)
+      q = CatalogQuery(0, basePosition, c.criterion.radiusLimits, c.criterion.magConstraints)
```

Comment:
This seems kind of smelly ... does the presence or absence of the zero determine whether it's a gems search or not? Or is there other information that does this? Would it make sense to have two constructors? `CatalogQuery.forGems(...)` etc.?